packages/twenty-front/src/modules/activities/emails/components/EmailThreadPreview.tsx

```diff
@@ -1,18 +1,28 @@
+import { css } from '@emotion/react';
 import styled from '@emotion/styled';
 
+import { EmailThreadNotShared } from '@/activities/emails/components/EmailThreadNotShared';
 import { CardContent } from '@/ui/layout/card/components/CardContent';
 import { grayScale } from '@/ui/theme/constants/colors';
 import { Avatar } from '@/users/components/Avatar';
 import { TimelineThread } from '~/generated/graphql';
 import { formatToHumanReadableDate } from '~/utils';
 
+const dynamicStyle = ({
+  visibility,
+}: {
+  visibility: 'metadata' | 'subject' | 'share_everything';
+}) => css`
+  cursor: ${visibility === 'share_everything' ? 'pointer' : 'default'};
+`;
+
 const StyledCardContent = styled(CardContent)`
```

Comment:
I was more thinking about this pattern
```
const StyledCardContent = styled(CardContent)<{ visibility: string }>`
  align-items: center;
  display: flex;
  gap: ${({ theme }) => theme.spacing(2)};
  height: ${({ theme }) => theme.spacing(12)};
  padding: ${({ theme }) => theme.spacing(0, 4)};
  cursor: ${({ visibility }) =>
    visibility === 'share_everything' ? 'pointer' : 'default'};
`;
```
You can ignore if it's not used this way in the rest of the repo but it was my assumption
@bosiraphael